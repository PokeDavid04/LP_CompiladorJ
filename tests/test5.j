square =: *:
square 1 2 3 4

mod2 =: 2 | ]
mod2 i. 4

eq0 =: 0 = ]
parell =: eq0 @: mod2
parell i. 6


square =: *:
square 1 + i. 3  

mod2 =: 2 | ]
eq0 =: 0 = ]

eq0 mod2 i. 6  

parell =: eq0 @: mod2
parell i. 6  

parell =: 0 = ] @: 2 | ]
parell i. 6

inc =: 1 + ]
test =: +/ @: inc @: i.
test 3 
