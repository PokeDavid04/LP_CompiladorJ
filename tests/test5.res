square =: <asignada> 
1 4 9 16 

mod2 =: <asignada> 
0 1 0 1 

eq0 =: <asignada> 
parell =: <asignada> 
1 0 1 0 1 0 


square =: <asignada> 
1 4 9 

mod2 =: <asignada> 
eq0 =: <asignada> 

1 0 1 0 1 0 

parell =: <asignada> 
1 0 1 0 1 0 

parell =: <asignada> 
1 0 1 0 1 0 

inc =: <asignada> 
test =: <asignada> 
6 

