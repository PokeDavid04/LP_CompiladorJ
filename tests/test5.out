

square =: <funcion> 
1 4 9 16 

mod =: <funcion> 
0 1 0 1 

eq =: <funcion> 
